index.html

```diff
@@ -1,63 +1,205 @@
+      </p>
+      <h2 class="supporting-text element">LET’S CONNECT</h2>
+      <ul class="element" id="social-media-container">
+        <l1>
+          <a href="#">
+            <img class="icon" alt="Twitter's icon" src="./visuals/twitter-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="LinkedIn's icon" src="./visuals/linkedin-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="Medium's icon" src="./visuals/medium-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="GitHub's icon" src="./visuals/github-icon.png" />
+          </a>
+        </l1>
+        <l1>
+          <a href="#">
+            <img class="icon" alt="AngelList's icon" src="./visuals/angellist-icon.png" />
+          </a>
+        </l1>
+      </ul>
+    </div>
+  </section>
+  <section id="works">
+    <div class="work-card">
+      <img class="work-snapshot element" src="./visuals/tonic-snapshot.png" />
+      <div class="work-text-block element">
+        <h1 class="title" id="headline-title">
+          Tonic
         </h1>
-        <p class="paragraph element">
-          I’m a software developer! I can help you build a product, feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hesitate to contact
-          me.
-        </p>
-        <h2 class="supporting-text element">LET’S CONNECT</h2>
-        <ul class="element" id="social-media-container">
-          <l1>
-            <a href="#">
-              <img class="icon" src="./visuals/twitter-icon.png" />
-            </a>
-          </l1>
-          <l1>
-            <a href="#">
-              <img class="icon" src="./visuals/linkedin-icon.png" />
-            </a>
-          </l1>
-          <l1>
-            <a href="#">
-              <img class="icon" src="./visuals/m-icon.png" />
-            </a>
-          </l1>
-          <l1>
-            <a href="#">
-              <img class="icon" src="./visuals/github-icon.png" />
-            </a>
-          </l1>
-          <l1>
-            <a href="#">
-              <img class="icon" src="./visuals/love-peace-icon.png" />
-            </a>
-          </l1>
+        <ul class="client-role-year">
+          <li class="work-client">CANOPY</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-role">Back End Dev</li>
+          <span class="bullet">&#x2022</span>
+          <li class="work-year">2015</li>
         </ul>
+        <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-list">
+          <li class="tag">HTML</li>
+          <li class="tag">CSS</li>
+          <li class="tag">JavaScript</li>
+        </ul>
+        <a class="button" href="#">See project</a>
```

Comment:
Please consider using the same border for the button to make the styles exactly in accordance with the Figma design:-

### YOUR DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/235076909-27ec723f-8d4f-4696-9d14-b5c6df026ed1.png)


### FIGMA DESIGN:-

![image](https://user-images.githubusercontent.com/94223367/235076964-81688d21-1235-466d-a85d-219d98f22896.png)


## NOTE:-


This problem exists in all similar buttons so also consider reviewing them to match the styles with the design.